### ISSUE TITLE
Tushy/Vixen/Blacked/Deeper/Slayed not matching

### ISSUE THREAD

**Issue by 3charac** [PAhelper/PhoenixAdult.bundle#1760]
All the recently added videos for these sites will not match. Have tried title, actor, and date

**Comment by r3arsh** [comment]
weird seems to be working fine for me for all vixen sites

whats your search query look like?

can you provide logs?

**Comment by 3charac** [comment]
Since there is no sceneID on the sites, I have just been using title. It used to work, and I have several movies that have matched that way. If I go back to the ones that were previously matched, I cannot get them to work now. I have also tried using actress name, and date

Slayed - Surveillance
Slayed - Adriana Chechik 
Slayed - Scarlit Scandal
Slayed - 2021.08.19

or another example from Tushy:
Tushy - Arty Babe Gets Anal From Client
Tushy - Gigi Allens
Tushy - 2015.11.16

Would logs show what is happening? How would I go abouts getting them, should I just do a bunch of failed searches first?

**Comment by 3charac** [comment]
I just re-loaded PAHelper since you had luck with yours, and it seems to work. I will test more extensively with some other sites later tonight, thank you!

**Comment by captaincoitus** [comment]
> Since there is no sceneID on the sites, I have just been using title. It used to work, and I have several movies that have matched that way. If I go back to the ones that were previously matched, I cannot get them to work now. I have also tried using actress name, and date
> 
> Slayed - Surveillance Slayed - Adriana Chechik Slayed - Scarlit Scandal Slayed - 2021.08.19
> 
> or another example from Tushy: Tushy - Arty Babe Gets Anal From Client Tushy - Gigi Allens Tushy - 2015.11.16
> 
> Would logs show what is happening? How would I go abouts getting them, should I just do a bunch of failed searches first?

I used [Webite] - [Date] - [Pornstar] for Strike3 Network video and it works flawlessly every time. 
For example: Following video of Haley Reed from TushyRaw will be named as: 
https://www.tushyraw.com/videos/snack-time

Title for PA: TushyRaw - 21-09-22 - Haley Reed

**Comment by Leinaria** [comment]
I'm having the same issue, nothing is coming up at all. Other unrelated sites seem to work fine.

File Name: Vixen.18.09.06.Tori.Black.And.Little.Caprice.Sugar.Daddy.Sharing.XXX.1080p.HEVC.x265.PRT

Search Strings:
Vixen 2018-09-06 Tori Black and Little Caprice Sugar Daddy Sharing
Vixen - 2018-09-06 - Tori Black and Little Caprice - Sugar Daddy Sharing
Vixen - 2018-09-06 - Tori Black - Sugar Daddy Sharing
Vixen - 2018-09-06 - Tori Black
Vixen - Tori Black
Vixen Tori Black
Vixen - Sugar Daddy Sharing

[Plex Media Server Logs_2023-06-04_11-53-20.zip](https://github.com/PAhelper/PhoenixAdult.bundle/files/11646032/Plex.Media.Server.Logs_2023-06-04_11-53-20.zip)

**Comment by r3arsh** [comment]
> I'm having the same issue, nothing is coming up at all. Other unrelated sites seem to work fine.
> 
> File Name: Vixen.18.09.06.Tori.Black.And.Little.Caprice.Sugar.Daddy.Sharing.XXX.1080p.HEVC.x265.PRT
> 
> Search Strings: Vixen 2018-09-06 Tori Black and Little Caprice Sugar Daddy Sharing Vixen - 2018-09-06 - Tori Black and Little Caprice - Sugar Daddy Sharing Vixen - 2018-09-06 - Tori Black - Sugar Daddy Sharing Vixen - 2018-09-06 - Tori Black Vixen - Tori Black Vixen Tori Black Vixen - Sugar Daddy Sharing
> 
> [Plex Media Server Logs_2023-06-04_11-53-20.zip](https://github.com/PAhelper/PhoenixAdult.bundle/files/11646032/Plex.Media.Server.Logs_2023-06-04_11-53-20.zip)

Can't check logs right now but have you downloaded the latest commit of the agent? Downloading and updating to the latest seems to be fixing it as Strike3Network/Vixen made some changes a few months ago.

**Comment by Leinaria** [comment]
I was using the one from 5/3/2023, but just installed the most recent one from 6/1/2023 (I wish there was some way to auto-update it). 

I'm still getting nothing at all coming up as search results for Vixen or Tushy.

**Comment by r3arsh** [comment]
> I just installed the most recent one from 6/1/2023 (I wish there was some way to auto-update it).
> 
> I'm still getting nothing at all coming up as search results for Vixen or Tushy.
> 
> I restarted the server just to make sure, still nothing coming up in the search.

Hey, sorry for the delay.

Checked your logs and noticed that there's no phoenixadult logs. Is your library agent set to 'PhoenixAdult'?

**Comment by Leinaria** [comment]
Yes, it is - Scanner: Plex Video Files, Agent: PhoenixAdult. Do I need to enable the Debug option? Or some other option?

**Comment by r3arsh** [comment]
> Yes, it is - Scanner: Plex Video Files, Agent: PhoenixAdult. Do I need to enable the Debug option? Or some other option?

Yeah please enable debug on the agent.

**Comment by Leinaria** [comment]
Okay, I enabled debug on the agent and then tried to do a search for this video:
Vixen - 2018-07-18 - Kenna James and Honey Gold - My Hollywood Threesome
Using various variations on the above search string.

Here are updated logs.
[Plex Media Server Logs_2023-06-10_15-47-42.zip](https://github.com/PAhelper/PhoenixAdult.bundle/files/11713192/Plex.Media.Server.Logs_2023-06-10_15-47-42.zip)

**Comment by Leinaria** [comment]
It looks like all these sites have switched to new Cloudflare DDoS protection, which may be what's causing the issue.

**Comment by wuyuzheyan** [comment]
vixen open a new site milfy

**Comment by XXXeraph** [comment]
All links (models, search, etc) go to a paywall now.  Suggest redirecting this to data18 scenes
